modules/i3s/src/lib/parsers/parse-i3s-tile-content.ts

```diff
@@ -274,7 +286,7 @@ function normalizeAttribute(attribute: I3SMeshAttribute): I3SMeshAttribute {
   return attribute;
 }
 
-function parseHeaders(tileset: I3STilesetHeader, arrayBuffer: ArrayBuffer) {
+function parseHeaders(tileset: I3STilesetOptions, arrayBuffer: ArrayBuffer) {
```

Comment:
- If this is now an options object and not a tileset instance, can we call the parameter `options`
- The normal parameter order is `(arrayBuffer, options)`